File: practicas/src/practica3/services/impl/OccasionalBuyer.java
package practica3.services.impl;

import practica3.services.WelcomeUser;

import javax.swing.*;

public class OccasionalBuyer implements WelcomeUser {

    @Override
    public void welcomeUser(String name) {
        JOptionPane.showMessageDialog(null, name + " you have successfully registered\n COMING SOON: web catalog of technology products.");
    }
}

File: ParcialAnalisis/Services/Imp/User/OccasionalBuyer.py
from Services.userServices.UserType import UserType

'''
This class represents an user type
'''


class OccasionalBuyer(UserType):

    def __init__(self):
        super().__init__()

    def sayHello(self):
        print("you have successfully registered\n COMING SOON: web catalog of technology products.")
